src/Neo/SmartContract/Native/OracleContract.cs

```diff
@@ -202,7 +204,7 @@ private async ContractTask Request(ApplicationEngine engine, string url, string
             if (urlSize > MaxUrlLength)
                 throw new ArgumentException($"The url bytes size({urlSize}) cannot be greater than {MaxUrlLength}.");
 
-            var filterSize = filter.GetStrictUtf8ByteCount();
+            var filterSize = filter is null ? 0 : filter.GetStrictUtf8ByteCount();
```

Comment:
Is this the modification that solved?

The other ones look like style as I see here

Reply:
Yes